internal/model/user_server.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/edulinq/autograder/internal/util"
 )
 
-var SERVER_USER_ROW_COLUMNS []string = []string{"email", "name", "server-role", "salt", "password", "tokens", "course-info"}
+var SERVER_USER_ROW_COLUMNS []string = []string{"email", "name", "server-role", "type"}
```

Comment:
I might have misunderstood your question in discord. But we don't care to print out the type of the user info in column form (every user in a users-list will be print "server" in this column, which isn't helpful).

Instead, we want to see the email, name, server-role, and courses.

Once you print out courses from the user info struct in TSV format, you will why I suggested the `--normalize` option.

Also, I don't like that model defines the expected columns for a server user. The past variable supports printing all of the server user information (the unsafe version of user info).

Your header definition is clearly geared to support the API safe representation of user information. But, the model package is below the API package and should not define the rows for the API structs.

I would suggest adding a new variable to the api/core/user.go file. You might want to leave the old one alone and add a note that this is an unsafe representation of user info (that includes passwords and tokens) and should be handled with care.